tests/let.cfg

```diff
@@ -0,0 +1,5 @@
+let arbtt = current window $title =~ m/arbtt/ in {
+    $arbtt && $time < 14:00 ==> tag arbtt-morning,
+    $arbtt && $time > 14:00 ==> tag arbtt-afternoon
+}
+
```

Comment:
Good job writing test cases. Can you extend it with a test where the `let`-bound condition fails, but some rule inside the body of the `let` does not use `$arbtt` and hence succeeds?  (To test that the body of a `let` thing is executed in any case.)

Reply:
Done